app/src/main/shared/styles/buttons.scss

```diff
@@ -1,28 +1,18 @@
+@import "bootstrap";
+
 button, .button {
-    display: inline-block;
-    border: none;
-    background-color: rgb(0, 173, 239);
-    color: white;
-    font-family: inherit;
-    font-size: 12pt;
+    @include button-variant(rgb(0, 173, 239), rgb(200, 200, 200));
+    @include button-size(5px, 20px, 12px, 24px,  0);
```

Comment:
Can we actually make the font size and line-height 1em and 2em? so `@include button-size(5px, 20px, 1em, 2em,  0);`